php/class-wp-cli.php

```diff
@@ -210,30 +210,36 @@ public function assign_coauthors( $args, $assoc_args ) {
 	 * @since 3.0
 	 *
 	 * @subcommand assign-user-to-coauthor
-	 * @synopsis --user_login=<user-login> --coauthor=<coauthor>
+	 * @synopsis --user_login=<user-login> --user_id=<user-id> --coauthor=<coauthor>
 	 */
 	public function assign_user_to_coauthor( $args, $assoc_args ) {
 		global $coauthors_plus, $wpdb;
 
 		$defaults = array(
-				'user_login'        => '',
+				'user_login'        => NULL,
+				'user_id'           => NULL,
 				'coauthor'          => '',
 			);
 		$assoc_args = wp_parse_args( $assoc_args, $defaults );
 
-		$user = get_user_by( 'login', $assoc_args['user_login'] );
+		$user_id = $assoc_args['user_id'];
```

Comment:
@RobJS I think the sensible way to check for int user IDs is to parse it properly here, which I probably needed to do anyway.

```suggestion
		$user_id = intval($assoc_args['user_id']);
```